ci/setup-build-test-gdk.ps1

```diff
@@ -5,7 +5,10 @@ param(
   [string] $target_platform = "Win64",
   [string] $build_home = (Get-Item "$($PSScriptRoot)").parent.parent.FullName, ## The root of the entire build. Should ultimately resolve to "C:\b\<number>\".
   [string] $unreal_path = "$build_home\UnrealEngine",
-  [string] $testing_project_name = "StarterContent" ## For now, has to be inside the Engine's Samples folder
+  [string] $test_repo_branch = "master",
```

Comment:
Would be good to be able to override this with an env_var, so we can test changes to the test project also. Unless that's being handled by the test project CI?

Reply:
Agreed, that's indeed helpful: The variable can now be overridden by defining TEST_REPO_BRANCH.